### PR TITLE
tagging: add dnd feature; collect/geotagging: add link to locations

### PR DESCRIPTION
**Pull request by phweyland**
Feel free to correct the English approximations.

Note: using the same editor settings as for dt, I get a lot of differences due to space trimming (this time I've removed them). Could that be also a rule for dtdocs ? Or do you mind if I submit my changes including those ?

### CODE REVIEW

File: content/module-reference/utility-modules/shared/tagging.md [hunk 1]
```diff
@@ -125,8 +125,12 @@ The "edit tag..." operation allows you to change the name of a tag, though you c
...
- 
-The "rename path..." operation is only available in the _tree_ view mode, and it shows the number of tagged images which would be impacted by a change to the name of this tag or category. The rename path window lets the user change the full path of tag, including the catagories to which it belongs (categories can be specified using the pipe "|" symbol). This operation is powerful, but please take care as it can have a significant impact on the metadata of your images. The operation is aborted if the requested change causes a conflict with an existing tag.
+
+The "rename path..." operation is only available in the _tree_ view mode, and it shows the number of tagged images which would be impacted by a change to the name of this tag or category. The rename path window lets the user change the full path of tag, including the categories to which it belongs (categories can be specified using the pipe "|" symbol). This operation is powerful, but please take care as it can have a significant impact on the metadata of your images. The operation is aborted if the requested change causes a conflict with an existing tag.
```
Comment (paperdigits) [inline content/module-reference/utility-modules/shared/tagging.md:129]: > lets the user change the full path of tag

should be: lets the user change the full path of the tag

the pipe symbol shouldn't use quotemarks, but should use the markdown for code, 
```
`|`
```

File: content/module-reference/utility-modules/shared/tagging.md [hunk 2]
```diff
@@ -125,8 +125,12 @@ The "edit tag..." operation allows you to change the name of a tag, though you c
...
+
+A quick way to organize the tag structure is the drag and drop. In the _tree_ view mode, you can drag whatever tag (node or leave) and drop it on top of another one (node or leave). The first one and its descendants if any become descendants of the second one. Dragging over a node automatically opens the node.
```
Comment (paperdigits) [inline content/module-reference/utility-modules/shared/tagging.md:131]: >A quick way to organize the tag structure is the drag and drop.

should be: A quick way to organize the tag structure is to drag and drop the tags

> you can drag whatever tag

should be: you can drag any tag

Additionally, what is a *leave* tag?

> The first one and its descendants if any become descendants of the second one.

should be: The first tag and its descendants, if any, become descendants of the second tag.
.

File: content/module-reference/utility-modules/shared/tagging.md [hunk 3]
```diff
@@ -125,8 +125,12 @@ The "edit tag..." operation allows you to change the name of a tag, though you c
...
+To place a tag at the root level, drag it onto the top of the tagging window to make appear a temporary record "drop to root" where you can drop it.
```
Comment (paperdigits) [inline content/module-reference/utility-modules/shared/tagging.md:132]: Can this be shorter?

Such as: To place a tag at the root level, drag it onto the top of the tagging window.

File: content/module-reference/utility-modules/shared/tagging.md [hunk 4]
```diff
@@ -125,8 +125,12 @@ The "edit tag..." operation allows you to change the name of a tag, though you c
...
+As for "rename path..." the operation is aborted if the requested change causes a conflict with an existing tag.
```
Comment (paperdigits) [inline content/module-reference/utility-modules/shared/tagging.md:133]: Perhaps something like: If a "rename path..." operation will cause conflicts with an existing tag, the operation is aborted.

**Comment by paperdigits** [comment]
@phweyland thanks! Please see my edits. Always open for discussion about any change.

File: content/module-reference/utility-modules/shared/tagging.md (hunk 2 above)
Comment (phweyland) [reply content/module-reference/utility-modules/shared/tagging.md:131]: > Additionally, what is a leave tag?

In a tree we have the nodes (every element part of the path) which can be closed or opened, and the leaves, the extremities of the paths.

File: content/module-reference/utility-modules/shared/tagging.md (hunk 3 above)
Comment (phweyland) [reply content/module-reference/utility-modules/shared/tagging.md:132]: The tagging window has no room to drop a tag to root, the root not being represented.
The trick here is to make appear this "drop to root" tag to allow this operation. 
And to make appear this temporary tag, the user needs to drag a tag, move the mouse to the top of the window. Only then the user can release his tag on top of it and move effectively a tag on the root of the tree. Once done the temporary "drop to root" tag disappears.

File: content/module-reference/utility-modules/shared/tagging.md (hunk 1 above)
Comment (phweyland) [reply content/module-reference/utility-modules/shared/tagging.md:129]: :+1:

File: content/module-reference/utility-modules/shared/tagging.md (hunk 4 above)
Comment (phweyland) [reply content/module-reference/utility-modules/shared/tagging.md:133]: :+1:

File: content/module-reference/utility-modules/shared/tagging.md (hunk 2 above)
Comment (johnny-bit) [reply content/module-reference/utility-modules/shared/tagging.md:131]: It should've been "leaf" tag then ;)

Comment (phweyland) [reply content/module-reference/utility-modules/shared/tagging.md:131]: You are right ... :confused:

File: content/module-reference/utility-modules/shared/tagging.md (hunk 3 above)
Comment (phweyland) [reply content/module-reference/utility-modules/shared/tagging.md:132]: If you think this is enough, let's do it.

**Comment by elstoc** [comment]
@phweyland is this a new feature or just an update to the docs? Just trying to work out whether this should go in the 3.4.1 docs or just 3.6.

**Comment by phweyland** [comment]
this should go in the 3.4.1 docs. Thanks

**Comment by elstoc** [comment]
Ok thanks

**Pull request merged**